src/types/predicates.ts

```diff
@@ -24,8 +22,8 @@ export type IsNumber<T> = T extends number ? True : False;
 export type IsString<T> = T extends string ? True : False;
 export type IsFunction<T> =
     Or<
-        T extends AnyFunc ? True : False,
-        T extends Function ? True : False>; // tslint:disable-line
+        T extends Function ? True : False,
```

Comment:
The find/replace made this condition redundant.

Reply:
Merp. Good catch. Not sure how I missed it on my final scan